app/assets/stylesheets/application.css

```diff
@@ -136,6 +143,10 @@ img {
   margin-left: 48px;
 }
 
+.color-blue {
+  color: blue !important;
```

Comment:
Remove `!important` from here

Reply:
Need to use this color-blue class in button-click.js file on click function. cannot use id or double class here. then couldn't find way to use this color without using important!